apps/common/src/python/mediawords/languages/en/en_stop_words.txt

```diff
@@ -4178,506 +801,195 @@ there's
 there've
 thereafter
 thereby
+thered
 therefore
 therein
+therell
+thereof
+therere
 theres
+thereto
 thereupon
-thermal
+thereve
 these
 they
 they'd
 they'll
 they're
 they've
-thick
-thickness
+theyd
+theyll
+theyre
+theyve
 thin
 thing
 things
-think
-thinking
-thinks
 third
 thirty
 this
 thorough
 thoroughly
 those
+thou
 though
-thought
-thoughts
+thoughh
 thousand
-thousands
-threat
-threatened
-threatening
 three
-threw
-throat
+throug
 through
 throughout
-throw
-thrown
 thru
-thrust
-thursday
 thus
-thyroid
-tie
-tied
-tight
+til
 till
-time
-times
-tiny
 tip
-tire
-tired
-tissue
-title
+tis
 to
-toast
 today
-toes
 together
-told
-tomorrow
-tone
-tones
-tongue
-tonight
-tons
 too
 took
-tool
-tools
-tooth
 top
-torn
-tossed
-total
-totally
-touch
-touched
-tough
-tour
-tournament
 toward
 towards
-town
-towns
-trace
-track
-tractor
-trade
-traders
-trading
-tradition
-traditional
-traditions
-traffic
-tragedy
-tragic
-train
-trained
-training
-transfer
-transferred
-transformation
-transformed
-transition
-transportation
-trap
-travel
-traveled
-treat
-treated
-treatment
-tree
-trees
-trembling
-tremendous
-trend
-trends
-trial
-trials
-triangle
-tribute
 tried
 tries
-trim
-trip
-trips
-triumph
-troops
-trouble
-troubled
-troubles
-truck
-trucks
-true
-truly
-trust
-truth
 try
 trying
-tsunami
-tube
-tubes
-tuesday
-turn
-turned
-turning
-turns
+twas
+twelve
 twenty
-twenty-five
 twice
 two
-type
-types
-typical
 u
-ugly
-ultimate
-un
-unable
-uncertain
-uncle
-unconscious
 under
-underground
-underlying
 underneath
-understand
-understanding
-understood
-undoing
-undoubtedly
-uneasy
-unexpected
-unfortunate
-unfortunately
-unhappy
-uniform
-union
-unions
-unique
-unit
-units
-unity
-universal
-universe
-universities
-unknown
 unless
 unlike
-unlikely
 until
 unto
-unusual
 up
-update
 upon
-upper
-upstairs
-upward
-upwards
-urban
-urge
-urged
-urgent
-url
+ups
 us
```

Comment:
Idea for a fun flamewar in the future: `us` is definitely a stopword but `US` is an abbreviation of a country which could be considered a stopword in the US media but not so much in other English-speaking countries. So perhaps those stopwords could be made case-sensitive at some point?